subprojects/packagefiles/gflags/meson.build

```diff
@@ -0,0 +1,49 @@
+project(
+  'gflags',
+  'cpp',
+  'c',
+  version: '2.2.2',
+  default_options: ['default_library=static'],
+)
```

Comment:
Looks like the wrap needs `meson_version: '>= 0.50'` to use the `install` arg in `configure_file()`.